pkg/prom/queries.go

```diff
@@ -0,0 +1,483 @@
+package prom
```

Comment:
Is this necessary within this PR? Happy to discuss changes here in a separate one!

Reply:
Thank you!